AboutDoc/Documentation_About.rst

```diff
@@ -0,0 +1,250 @@
+It is written in Python, and also used in other environments. 
+It is licensed under the BSD license.
+It can also export files for translation.
+
+Generating a local copy of the manual requires only the Python-based Sphinx documentation system and the Sphinx RTD theme.  
+They can be installed  using most package managers, or via 
+
+.. code-block:: console
+
+  pip3 install sphinx sphinx-rtd-theme
+
+Anyone who would like to contribute and needs help with installing and using Sphinx can ask for support in the issues tracker.
+
+Tutorial video:	https://www.youtube.com/watch?v=ouHVkMo3gwE
+
+ReST Basic Syntax
+-----------------
+`List of basic syntax </Documentation_RestSyntax.rst>`_  in Openshot documentation.  
+
+- Some explanation here:  https://hyperpolyglot.org/lightweight-markup
+- or here: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html
+- Video tutorials here:  https://www.youtube.com/results?search_query=restructuredtext+tutorial
+
+Software 
+
+- Notepad++ ReST template:	https://github.com/steenhulthin/reStructuredText_NPP
+- Linux: 
+
+File naming and directory structure
+-----------------------------------
+
+| Files are stored in \docs
+| File names may not contain spaces. Spaces in filenames cause problems with Sphinx. 
+| For multi-word filenames, ReST documentation filenames are separated by an underscore ( _ ).
+| Image filenames are separated by a dash ( - ). 
+| Images are named after the file they belong to, and sorted in the doc\\images subdirectory. 
+
+.. NOTE: Or maybe not, see Images
+
+Translations go into their own directory. 
+
+File structure
+--------------
+
+Every file starts with 5 hidden segments (See `template <Template.rst>`_ )
+
+- Copyright notice
+- Openshot description
+- Openshot disclaimer
+- License notice
+- Referral anchor for the title
+
+Followed by the title with double underlining (with == )
+Then a short description of what the chapter will be about, and why the reader should care (or skip). 
+Subtitles are single underlined ( -- )
+
+Sentences should be written one sentence per line, and do not need to end on a space.
+The markup language then flows them all together into paragraphs when it generates the formatted docs.
+You can also break at other logical points, like after a comma in a longer sentence,
+or before starting an inline markup command.
+It is a guideline, not a rule.
+This tends to be a pretty good fit for any sort of written prose, when it is in a markup language like ReST and managed in a version control system.
+
+There are four reasons for this:
+
+- Writing that way, there is no worrying about line length or when to wrap. 
+- It encourages shorter, simpler sentences which is a good thing when writing docs. 
+- The diffs when changes are submitted also tend to be more readable and focused. 
+- Lines are easier to translate and less likely to be changed. 
```

Comment:
Again, nix all mentions of translation please. Especially since this one also isn't true, or at least doesn't matter. When Sphinx outputs translation files, it flows sentence-per-line blocks back together and exports the entire paragraph as a single translation string.